File: AngularAppDataServer/AngularAppDataServer/Models/News.cs
using System;

namespace AngularAppDataServer.Models
{
    public class News
    {
        public int Id { get; set; }
        public DateTime Date { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }
        public NewsType Type { get; set; } = NewsType.Type0_None;

        public News(int Id, DateTime Date, string Title, string Body, NewsType Type = NewsType.Type0_None)
        {
            this.Id = Id;
            this.Date = Date;
            this.Title = Title;
            this.Body = Body;
            this.Type = Type;
        }
    }
}

File: block8/src/model/News.ts
import '@angular/common/locales/global/ru';
import { TypeNews } from './TypeNews';

export class News {
    public id:number = 0;
    public date:Date;
    public title:string = "";
    public body:string = "";
    public type:TypeNews = TypeNews.Type0_None    

    constructor(id:number, date: Date, title: string, body:string, type:TypeNews = TypeNews.Type0_None)
    {
        this.id = id;
        this.date = date;
        this.title = title;
        this.body = body;
        this.type = type;
    }
  }